### PR TITLE
Flex 2850 show template schedule can be used as basis for new schedule

### PR DESCRIPTION
**Pull request by mizendooren**
Kunnen jullie deze bekijken? dank

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://cla.opensmartgridplatform.org:80/OSGP/Integration-Tests?pullRequest=212) <br/>All committers of the pull request should sign our Contributor License Agreement in order to get your pull request merged.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: mizendooren<br/>:x: web-flow

File: cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/Defaults.java [hunk 1]
```diff
@@ -207,7 +207,13 @@
     public static final String LIGHTMEASUREMENT_LASTMESSAGE = "";
     public static final Integer SMS_INDEX = 1;
 
-    public static final Boolean DEFAULT_BYPASS_RETRY = false;
+    public static final boolean DEFAULT_BYPASS_RETRY = false;
```
Comment (Joep-Simons) [inline cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/Defaults.java]: Why was this changed from Boolean to boolean?

**Comment by Joep-Simons** [comment]
Please do not use the Github browser editor to change files, because now the CLA cannot be signed. There is no need to create a new PR and this can be merged, but keep it in mind for future reference.

File: cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/Defaults.java (hunk 1 above)
Comment (MartijnSipsCGI) [inline cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/Defaults.java]: This doesn't matter. Boolean gives you more methods, but boolean uses way less memory (in relation to each other). In the tests this doesn't matter that much, because we're only using this variable in equasions......
